applications/serial_lte_modem/src/main.c

```diff
@@ -339,6 +341,8 @@ void handle_mcuboot_swap_ret(void)
 
 int start_execute(void)
 {
+	k_sem_take(&entered_slm, K_SECONDS(1));
```

Comment:
What's this for?

Reply:
It was needed to ensure that the carrier library had initialized before continuing, but it is no longer needed. Removed.